src/data/evm_assets.json

```diff
@@ -1351,6 +1351,119 @@
         }
       ]
     },
+    {
+      "id": "frxeth-wei",
```

Comment:
do we have a logo svg in the repo for this?

Reply:
here : https://github.com/axelarnetwork/axelar-docs/blob/main/public/images/assets/frxeth.svg